core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -933,14 +933,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
-  public static final PropertyKey MASTER_HEARTBEAT_INTERVAL_MS =
-      new Builder(Name.MASTER_HEARTBEAT_INTERVAL_MS)
-          .setAlias(new String[]{"alluxio.master.heartbeat.interval.ms"})
-          .setDefaultValue("10sec")
+  public static final PropertyKey MASTER_MASTER_HEARTBEAT_INTERVAL_MS =
+      new Builder(Name.MASTER_MASTER_HEARTBEAT_INTERVAL_MS)
+          .setAlias(new String[]{"alluxio.master.master.heartbeat.interval.ms"})
```

Comment:
so `MASTER_MASTER_HEARTBEAT_INTERVAL_MS` will replace `MASTER_HEARTBEAT_INTERVAL_MS`, right?
In this case,  to keep backward compatible, you can do:
```
.setAlias(new String[]{"alluxio.master.heartbeat.interval.ms", "alluxio.master.heartbeat.interval"}) 
```
We don't need `"alluxio.master.master.heartbeat.interval.ms"` here as it is never used before

Reply:
The original MASTER_HEARTBEAT_INTERVAL_MS is used for master and worker heartbeat, I will put the Alias in the MASTER_WORKER_HEARTBEAT_INTERVAL_MS, thanks